library/template/cache.php

```diff
@@ -94,8 +94,7 @@ public function loadString($content)
      */
     public function render(array $data = array())
     {
-        $this->getDelegate()->render($data);
-        return $this;
+        return $this->getDelegate()->render($data);
```

Comment:
Yup, if one is intending to call `render()` on the template object, that is decorated, one should expect to get the rendered value back.